saos-webapp/src/main/webapp/WEB-INF/view/common/saosLogo.jsp

```diff
@@ -0,0 +1,13 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+
+
+<h1>
```

Comment:
I don't think the h1 should be around logo. It's not the case for which h1 has been designed for, don't you think?